morph-all-view-graphql-files.js

```diff
@@ -154,7 +154,7 @@ ${isUsingDataOnSubmit ? '  let onSubmit = useDataOnSubmit(props, data)' : ''}
   return (
     <DataProvider
       context="${context}"
-      value={data}
+      value={data?.${context}}
```

Comment:
is this to make them work as they're now defined?

Reply:
this is because a query response will include the top level name which is used for creating the context name. e.g.

```
query {
  threads: vaxiom_threads {
    id
    name
  }
}
```

the response of this query will be something like this:

```
{
  threads: [{...}]
}
```

but now there is no point storing the context name key as a value so this is why it will access the value like this `data.threads`. let me know if you think there is anything that can be improved here, thanks

Reply:
Aah gotcha! Let's try with that